### PR TITLE
Fixes #1244, Fixed Bookmark snackbar.

### LINKED ISSUE #1244: Illogical flow while removing a bookmark
**Actual Behaviour**

When we are removing a bookmark, a snackbar appears with an `Undo` button. This happens in every Activity or Fragment.

**Expected Behaviour**

In fragments or activities other than `Bookmarks` the snackbar is not needed. Even if the user removes a bookmark he can add it again because the session is still on the screen.

The snackbar should only appear if the action is performed in bookmark fragment (because if the user removes bookmark the session disappears from the list and action can only be undone using a snackbar)

@the-dagger 
I am sending a PR.

### PR DESCRIPTION
**Pull request by free4murad**
Fixes #1244 

Changes: Snackbar with undo appears only if the session is removed from BookmarksFragment.

### CODE REVIEW

**Comment by free4murad** [comment]
@the-dagger @mananwason 
Please review the PR.

**Comment by harshitagupta30** [comment]
@free4murad Can you please update your commit message according to [guidelines](https://github.com/fossasia/open-event-android/blob/development/docs/commitstyle.md)? :+1:

File: android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java [hunk 1]
```diff
@@ -196,22 +196,26 @@ public void onClick(View v) {
...
-                                        bookmarksListChangeListener.onChange();
+                    if("MainActivity".equals(context.getClass().getSimpleName())){
+                        Snackbar.make(v, R.string.removed_bookmark, Snackbar.LENGTH_LONG)
+                                .setAction(R.string.undo, new View.OnClickListener() {
+                                    @Override
+                                    public void onClick(View v) {
+                                        dbSingleton.addBookmarks(session.getId());
+                                        holder.sessionBookmarkIcon.setImageResource(R.drawable.ic_bookmark_white_24dp);
+                                        WidgetUpdater.updateWidget(context);
+                                        if(bookmarksListChangeListener != null){
```
Comment (harshitagupta30) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: Can you please reformat this line?

Comment (free4murad) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: Sure

**Comment by free4murad** [comment]
@the-dagger 
please review and merge

**Comment by mariobehling** [comment]
Please provide screenshots. I am not entirely clear about this issue.

**Comment by free4murad** [comment]
What I have implemented in this is if the user removes a session from bookmarks list in activity other than  BookmarksFragment the a snackbar with no action will appear. (Undo action is not needed because the user can bookmark the session there only)
But if the user removes a session from bookmarks list in BookmarksFragment activity the sandbar will appear with undo action.

**Comment by mananwason** [comment]
@free4murad Please resolve conflicts

**Comment by free4murad** [comment]
In a minute.

**Comment by free4murad** [comment]
@mananwason Done

**Comment by free4murad** [comment]
@the-dagger @mananwason 
please review.

**Pull request merged**